tracking/src/main/java/org/motechproject/nms/tracking/service/CacheService.java

```diff
@@ -0,0 +1,5 @@
+package org.motechproject.nms.tracking.service;
+
+public interface CacheService {
```

Comment:
I'll look into it this morning. And also will check if we can cache
repository lookups directly.

On Wed, Sep 16, 2015 at 4:53 PM, Rob LaRubbio <notifications@github.com>
wrote:

> In
> tracking/src/main/java/org/motechproject/nms/tracking/service/CacheService.java
> <https://github.com/motech-implementations/mim/pull/590#discussion_r39699290>
> :
>
> > @@ -0,0 +1,5 @@
> > +package org.motechproject.nms.tracking.service;
> > +
> > +public interface CacheService {
>
> Or if we keep the new service, instead of calling it CacheService I think
> we should rename it the EntityServiceWrapper and not expose it. So instead
> of placing it in the service package we should either see if we've done
> something like this before and copy that, or create a new package
> internal/service so it isn't exposed via osgi.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/590/files#r39699290>.
>



-- 
Frank Huster
Mobile Health Innovations
Grameen Foundation
+1.206.325.6690 x201